elastic_record.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'arelastic', '>= 0.7.0'
   s.add_dependency 'activemodel'
+  s.add_dependency 'oj'
```

Comment:
I would like to see this removed. Something like `ElasticRecord.json_parser = :oj`.